documentation/Functions.md

```diff
@@ -297,18 +297,41 @@ will run 'npm install' in the directory 'my-thai-star/angular'
 ***
 
 ### dockerCompose <a name="dockerCompose"></a>
+This function is used for running Docker Compose. For more information on Docker Compose click [here](https://docs.docker.com/compose/) 
+
 #### parameter 
-1. Path to the directory where the docker-compose.yml file is located, relative to workspace.
-2. Assertion information. Only needed for the console runner to check if the server was started properly.
+This function has consist of two parameters.
+
+1. First parameter:
+    - **Required**
+    - **Type**- String
+    - **Description**- This is the path to the directory where the docker-compose.yml file is located, relative to workspace.
+
+2. Second parameter:
+    - **Required**
+    - **Type**- JSON Object
+    - **Description**- In this parameter you need to give assertion information. This parameter is only needed for the console runner to check if the server started properly.
+
+## Assertion Information
+You need three attributes in second parameter for assertion information. 
+
+1. startupTime = Time in seconds to wait before checking if the server is running
+2. port: Port on which the server is running
+3. path: The URL path on which is checked if the server is running
+
 #### example 
 dockerCompose("my-thai-star", { "startupTime": 600, "port": 8081, "path": "" })
 
-##### Assertion information
-startupTime = Time in seconds to wait before checking if the server is running
-port: Port on which the server is running
-path: The URL path on which is checked if the server is running
+Note:
+1. If the tutorial needs to be tested on the console environment, you have to specify a port.
 
-If the tutorial should be tested on the console environment, you have to specify a port.
+2. No background script will run. 
+
+3. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
+
+4. Docker must be installed. In the Katacoda environment, Docker is already preinstalled.
+
+5. TO use this function, devonfw-ide must be installed. 
```

Comment:
devonfw-ide has not to be installed. Only Docker which is preinstalled in Katacoda as you mentioned above.